### PR TITLE
Remove extra sector question in business support flow

### PR DESCRIPTION
**Pull request by theseanything**
This merges the "Is your a business a nightclub, dancehall or adult entertainment venue?" question as another option within the pre-existing sector question. The sector question has also been refactored to make it clear that users can select multiple options.

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

File: test/integration/smart_answer_flows/business_coronavirus_support_finder_test.rb [hunk 1]
```diff
@@ -1,58 +0,0 @@
-require_relative "../../test_helper"
```
Comment (gclssvglx) [inline test/integration/smart_answer_flows/business_coronavirus_support_finder_test.rb:1]: I understand why this needs deleting - but it doesn't really relate to this PR.  Should it be a separate PR?

File: lib/smart_answer/calculators/business_coronavirus_support_finder_calculator.rb [hunk 2]
```diff
@@ -65,11 +64,11 @@ def initialize
       },
       lrsg_open: lambda { |calculator|
         calculator.business_based == "england" &&
-          calculator.restricted_sector == "no"
+          !calculator.sectors.include?("nightclubs_or_adult_entertainment")
```
Comment (gclssvglx) [inline lib/smart_answer/calculators/business_coronavirus_support_finder_calculator.rb:67]: Just a suggestion - would using `exclude?` be better here?  I know `exclude?` is defined in `ActiveSupport` - however I feel it reads better.

File: test/integration/smart_answer_flows/business_coronavirus_support_finder_test.rb (hunk 1 above)
Comment (theseanything) [reply test/integration/smart_answer_flows/business_coronavirus_support_finder_test.rb:1]: Yup - so because I had changed the flow I'd need to update these tests to make them pass. Figured if they are going to be deleted anyways thats the easier change to make.

File: lib/smart_answer/calculators/business_coronavirus_support_finder_calculator.rb (hunk 2 above)
Comment (theseanything) [reply lib/smart_answer/calculators/business_coronavirus_support_finder_calculator.rb:67]: ahh nice! I'll update that

File: test/integration/smart_answer_flows/business_coronavirus_support_finder_test.rb (hunk 1 above)
Comment (gclssvglx) [reply test/integration/smart_answer_flows/business_coronavirus_support_finder_test.rb:1]: No worries

File: lib/smart_answer/calculators/business_coronavirus_support_finder_calculator.rb (hunk 2 above)
Comment (gclssvglx) [reply lib/smart_answer/calculators/business_coronavirus_support_finder_calculator.rb]: Nice - reads much better 👍
